modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -92,29 +74,30 @@
 	if(is_banned_from(user, "emote"))
 		to_chat(user, span_warning("You cannot send subtle emotes (banned)."))
 		return FALSE
-	else if(user.client && user.client.prefs.muted & MUTE_IC)
+	else if(user.client?.prefs.muted & MUTE_IC)
 		to_chat(user, span_warning("You cannot send IC messages (muted)."))
 		return FALSE
 	else if(!subtler_emote)
 		subtler_emote = tgui_input_text(user, "Choose an emote to display.", "Subtler" , null, MAX_MESSAGE_LEN, TRUE)
-		if(subtler_emote && !check_invalid(user, subtler_emote))
-			var/list/in_view = get_hearers_in_view(1, user)
-			in_view -= GLOB.dead_mob_list
-			in_view.Remove(user)
-
-			for(var/mob/mob_in_view as anything in in_view)
-				if(!istype(mob_in_view))
-					in_view.Remove(mob_in_view)
-			var/list/targets = list(SUBTLE_ONE_TILE_TEXT, SUBTLE_SAME_TILE_TEXT) + in_view
-			target = tgui_input_list(user, "Pick a target", "Target Selection", targets)
-			switch(target)
-				if(SUBTLE_ONE_TILE_TEXT)
-					target = SUBTLE_DEFAULT_DISTANCE
-				if(SUBTLE_SAME_TILE_TEXT)
-					target = SUBTLE_SAME_TILE_DISTANCE
-			subtler_message = subtler_emote
-		else
+		if(!subtler_emote)
+			return FALSE
+		var/list/in_view = get_hearers_in_view(1, user)
+		in_view -= GLOB.dead_mob_list
+		in_view.Remove(user)
+
+		for(var/mob/mob_in_view in in_view)
+			if(!istype(mob_in_view))
```

Comment:
If you removed the `as anything`, then the `!istype()` isn't really necessary anymore, I feel like.